src/schnetpack/cli.py

```diff
@@ -155,13 +158,19 @@ def train(config: DictConfig):
 
     # Evaluate model on test set after training
     log.info("Starting testing.")
-    trainer.test(model=task, datamodule=datamodule)
+    trainer.test(model=task, datamodule=datamodule, ckpt_path="best")
 
-    # Print path to best checkpoint
-    log.info(f"Best checkpoint path:\n{trainer.checkpoint_callback.best_model_path}")
+    # Store best model
+    best_path = trainer.checkpoint_callback.best_model_path
+    log.info(f"Best checkpoint path:\n{best_path}")
 
+    log.info(f"Store best model")
+    best_task = type(task).load_from_checkpoint(best_path)
+    best_task.save_model(config.globals.model_path, do_postprocessing=True)
+    log.info(f"Best model stored at {os.path.abspath('best_model')}")
```

Comment:
would it make sense to write <config.globals.model_path> instead of <'best_model'> since <config.globals.model_path> can change depending on the configs?